Ghidra/Features/PDB/src/main/java/pdb/PdbSymbolServerPlugin.java

```diff
@@ -257,7 +257,7 @@ private void parseSymbolEnv(String envString) {
 		//    srv*[local cache]*[private symbol server]*https://msdl.microsoft.com/download/symbols
 		//    srv*c:\symbols*https://msdl.microsoft.com/download/symbols
 
-		if (!envString.startsWith("srv")) {
+		if (!envString.startsWith("srv") && !envString.startsWith("SRV")) {
```

Comment:
We may want to convert envString to all lower-case by invoking envString.toLowerCase. Then, startswith can work with changes.

Reply:
But someone on StackOverflow said that using toLowerCase is wrong because in some locales two different uppercase characters may have the same corresponding lowercase. I am not sure if it matters here, though.

https://stackoverflow.com/questions/19154117/startswith-method-of-string-ignoring-case#comment83993524_19154186

Reply:
This is good the way it is. While Microsoft's docs all use "srv", it seems there are a lot of forum posts which show "SRV" as well. I think we'd be splitting hairs trying to support users who use mixed case, e.g., Srv, SRv, etc.